guides/release/code-editors/index.md

```diff
@@ -25,10 +23,16 @@ and maintain consistent coding styles between different editors and IDEs.
 [Glimmer Templates Syntax](https://marketplace.visualstudio.com/items?itemName=lifeart.vscode-glimmer-syntax) -
 Syntax formatting for glimmer templates.
 
+[VSCode Glimmer](https://marketplace.visualstudio.com/items?itemName=chiragpat.vscode-glimmer) - 
```

Comment:
This gets you `hbs` highlighting in tests

Reply:
@NullVoxPopuli `lifeart.vscode-glimmer-syntax` have all this

Reply:
is that a meta package?

Reply:
Nope, standalone one